.travis.yml

```diff
@@ -3,11 +3,11 @@ sudo: false
 matrix:
     include:
     - python: "3.5"
-      env: DEPS="numpy=1.11.2 scipy=0.18.1 cython nose matplotlib"
+      env: DEPS="numpy=1.11.2 scipy=0.18.1 cython nose matplotlib scikit-learn=0.18.1"
     - python: "3.4"                                                                                         
-      env: DEPS="numpy=1.10.4 scipy=0.17 cython nose matplotlib"
+      env: DEPS="numpy=1.11.2 scipy=0.17 cython nose matplotlib scikit-learn=0.18.1"
```

Comment:
The main reason to support (and test) multiple version of dependencies is to reduce the chance of a dependency conflict (e.g. Package A depends on package `C-v1`, package B depends on `C-v2`, and you need A and B). I also use the latest numpy version, but in general we cannot assume that (e.g. in large legacy systems with a significant cost of upgrading). For instance,  scikit-learn will install `numpy-1.11` if it's not present but [it supports](http://scikit-learn.org/stable/install.html#installing-the-latest-release) any numpy versions starting from `1.6.1` (and also test several version in [Travis CI](https://travis-ci.org/scikit-learn/scikit-learn). Here we just test the 2 latest numpy version 0.10 for PY<3.5 and 0.11 for PY 3.5.

Point 2:  I was referring to the new Krige class you created in this PR.